CMakeLists.txt

```diff
@@ -30,6 +30,15 @@ option(MUSICA_BUILD_DOCS "Build the documentation" OFF)
 option(MUSICA_ENABLE_MICM "Enable MICM" ON)
 option(MUSICA_ENABLE_TUVX "Enable TUV-x" ON)
 
+set(MUSICA_SET_MICM_NUM_GRID_CELLS "1" CACHE STRING "Set the number of grid cells for MICM Solver")
```

Comment:
The number of grid cells is something that won't be know at compile time for the MUSICA library, if a host model is linking against an installed version of MUSICA.

Can this be passed as an argument to the MICM constructor?

Reply:
That's a good point. I made it to be passed as an argument to the MICM constructor.